hurumap/static/js/charts.js

```diff
@@ -37,6 +62,7 @@ function Chart(options) {
     chart.chartNullLabel = options.chartNullLabel || "N/A";
     chart.decimalPlaces = parseInt(options.chartDecimalPlaces) || 0;
     chart.tableDecimalPlaces = parseInt(options.chartDecimalPlaces) || 1;
+    chart.colorbrewer = Object.assign({}, options.colorbrewer, default_colorbrewer);
```

Comment:
Good idea but I'd rather we use `$.extend` so that we don't break any functionality. `Object.assign` doesn't work in any version of IE (which wazimap still supports me thinks).